pom.xml

```diff
@@ -41,7 +41,7 @@
         <guava.version>27.1-android</guava.version> <!-- From the docs: If you need support for JDK 1.7 or Android, use the Android flavor. -->
         <java-dogstatsd-client.version>2.9.0</java-dogstatsd-client.version>
         <jcommander.version>1.35</jcommander.version>
-        <log4j.version>2.13.2</log4j.version>
+        <log4j.version>2.12.1</log4j.version>
```

Comment:
A comment on this line would be good so the change doesn't happen again

Reply:
thanks, addressed 👍